apps/protocol-frontend-e2e/cypress.config.ts

```diff
@@ -11,13 +11,15 @@ module.exports = defineConfig({
     '../../dist/cypress/apps/protocol-frontend-e2e/screenshots',
   chromeWebSecurity: false,
   e2e: {
+    experimentalSessionAndOrigin: true,
```

Comment:
What does this configuration do?

Reply:
I was testing out the cy.session (which is still an experiemental feature)
I am gonna remove it